packages/framework/src/Model/Order/Order.php

```diff
@@ -1125,4 +1125,12 @@ public function getTrackingUrl(): ?string
             OrderMail::TRANSPORT_VARIABLE_TRACKING_NUMBER => $trackingNumber,
         ]);
     }
+
+    /**
+     * @return string
+     */
+    public function __toString(): string
+    {
+        return sprintf('Order with id: %s and number: %s', $this->id, $this->number);
```

Comment:
the __toString method will be mandatory for all entities in administration?

Reply:
If you don't implement `__toString()` method, you will see Entity with some hash in administration as title or in success/error message